apps/src/courseExplorer/courseExplorer.js

```diff
@@ -0,0 +1,40 @@
+// Code for the Course/Tools Explorer.  Include it once on the page and it will
+// handle both CourseExplorer and ToolsExplorer on the same page.
+
+var detailRowShowing = -1;
+var toolShowing = -1;
+
+module.exports.initCourseExplorer = function () {
+  $('.tool').click(function () {
+    var row = ($(this).data('row'));
+    var index = ($(this).data('index'));
+
+    if (toolShowing === -1) {
+      $('#detailrow-' + row).slideDown();
+      $('#toolsextra-' + index).fadeIn();
```

Comment:
Is all the JS here an indication that this UI should be created in React instead of haml?